src/routes/api/event/api.js

```diff
@@ -80,7 +80,10 @@ router.get(
       const query = res.locals.query;
 
       const count = await db.Event.count({ ...query, distinct: true });
-      const events = await db.Event.findAll(query);
+      const events = await db.Event.findAll({
+        ...query,       
+        order: [['highlighted', 'DESC']],
```

Comment:
Gaat dit goed met de order die mogelijk al in query zit?

Reply:
Goeie, die zal ik even moeten testen